apkid/rules/dex/obfuscators.yara

```diff
@@ -241,3 +241,19 @@ rule aamo_str_enc : obfuscator
   condition:
     1 of ($opcodes*) and all of ($a, $b)
 }
+
+rule kiwi_amazon : obfuscator
+{
+    meta:
+        description = "Kiwi encrypter"
```

Comment:
Is there a website for this or just some weird random obfuscator you've seen?

Reply:
No clue